client/my-sites/plans/jetpack-plans/product-store/index.tsx

```diff
@@ -57,6 +63,32 @@ const ProductStore: React.FC< ProductStoreProps > = ( {
 
 	const showJetpackFree = useShowJetpackFree();
 
+	const tabs = useMemo(
+		() =>
+			TABS.map( ( name ) => {
+				const titles = {
+					products: translate( 'Products' ),
+					bundles: translate( 'Bundles' ),
+				};
+
+				return {
+					name,
+					title: titles[ name ],
+				};
+			} ),
+		[ translate ]
+	);
+
+	useEffect( () => {
```

Comment:
Update the `view` query parameter as the user switches tab.

Reply:
Is there a way to avoid the URL changing on the initial page load? Right now, `/pricing` changes to `/pricing?view=bundles`

Reply:
Something like this inside that `useEffect` should do the job

```js
if ( ! didMount.current ) {
	didMount.current = true;
	return;
}
```

where
```js
const didMount = useRef( false );
```

Reply:
Added a fix in #69654